$OpenBSD$

Import compilation fix from r2510:

    Fix incorrect include header. Fix to ticket #245

Index: src/modules.cc
--- src/modules.cc.orig
+++ src/modules.cc
@@ -47,7 +47,7 @@ License along with this library; if not, see
 #undef interface
 #endif
 
-#include "error.h"
+#include "errors.h"
 #include "gpsim_interface.h"
 #include "stimuli.h"
 #include "symbol.h"
